src/ucp/core/ucp_context.c

```diff
@@ -444,6 +444,13 @@ static ucs_config_field_t ucp_context_config_table[] = {
    "directory.",
    ucs_offsetof(ucp_context_config_t, proto_info_dir), UCS_CONFIG_TYPE_STRING},
 
+  {"MEMTYPE_REG_NB_TYPES", "auto",
+   "Memory types that perform non-blocking registration by default:\n"
+   "page registration may be deferred until it is accessed by the CPU or a transport.\n"
+   "Allowed memory types: cuda, host.",
```

Comment:
@yosefe @brminich are that "default memory types" correct?

Reply:
no need to mention those, also `cuda` is not going to be supported